WooCommerce/Classes/ViewRelated/Orders/OrdersSplitViewWrapperController.swift

```diff
@@ -57,19 +57,59 @@ final class OrdersSplitViewWrapperController: UIViewController {
 }
 
 private extension OrdersSplitViewWrapperController {
-    func configureSplitView() {
-        let ordersNavigationController = WooTabNavigationController()
-        ordersNavigationController.viewControllers = [ordersViewController]
-
+    func showEmptyView() {
         let emptyStateViewController = EmptyStateViewController(style: .basic)
         let config = EmptyStateViewController.Config.simple(
             message: .init(string: Localization.emptyOrderDetails),
             image: .emptySearchResultsImage
         )
         emptyStateViewController.configure(config)
+        showSecondaryView(emptyStateViewController)
```

Comment:
This should be wrapped in a `UINavigationController`, or a `WooNavigationController`, before we show it in the split view. If it's not, `UISplitViewController` automatically wraps it, which sounds convenient, but the nav controller it gets wrapped in behaves a little strangely. For the empty view it might not cause any issues, but it's safer to avoid the default nav controller altogether.

Given that we should do this for anything we show in the secondary pane, we could do the wrapping in the `showSecondaryView` function which you've added.

Reply:
Good catch! Updated in https://github.com/woocommerce/woocommerce-ios/pull/11799/commits/240edf6b703bb819abf6d11537a9b1a0e760b159. I didn't move `UINavigationController` wrapping to `showSecondaryView` just to be more flexible, but we can consider moving the wrapping logic there if it's easy to miss it.